clients/admin-ui/src/features/common/table/grouped/GroupedTableBody.tsx

```diff
@@ -0,0 +1,128 @@
+import { Tbody, Td, Text, Tr } from "@fidesui/react";
```

Comment:
this is mostly copied from the original `DatamapTable` component, with just a few modifications to allow it to take props and some small customizations